### DISCUSSION TITLE
uosc的播放暂停动画怎么关掉

### DISCUSSION THREAD

**Discussion by yanayosihana** [hooke007/mpv_PlayKit#445] [Q&A 问答]
就按暂停和播放的时候视频中间出现的这个（如图），原版没有，uosc带的，有办法关掉吗
<img width="1280" alt="mmexport1712717266491" src="https://github.com/hooke007/MPV_lazy/assets/76032750/986ec405-e3d6-4ab5-a59f-46b2b193668d">

**Comment by hooke007** [comment]
你都知道是uosc带的了，为啥不去看看uosc的选项呢
